docs/src/pages/demos/text-fields/text-fields.md

```diff
@@ -86,6 +86,10 @@ Icons can be specified as prepended or appended.
 
 {{"demo": "pages/demos/text-fields/OutlinedInputAdornments.js"}}
 
+By default when using adornment in the begining `InputLabel` will be shrinked. In some cases this behaviour may not be desired. We can always restore default behaviour by setting `shrink='auto'`
```

Comment:
```suggestion
When using an InputAdornment at the start of the Input, by default the `InputLabel` will be shrunk. In some cases this behavior may not be desired. You can restore the default behavior by setting `shrink='auto'`.
```

> In some cases this behavior may not be desired.

An example might be helpful here. When would you want the label to overlap the icon?

Reply:
As I showed it in a screenshot of my specific component design:
![image](https://user-images.githubusercontent.com/437214/51038546-30e72380-15bc-11e9-8fd1-16a910176645.png)

Icon is not overlapping with label, therefore I don't want label to be shrunk by default. Does that makes sense?